### PR TITLE
Add notes for SIMD meeting on 2024-06-21

### PR DESCRIPTION
**Pull request by ppenzin**

### CODE REVIEW

**Comment by ppenzin** [comment]
@akirilov-arm - FYI, though I can't request a review from you, but I can give one.

File: simd/2024/SIMD-06-21.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+AK: That means that instead of 128-bit optimized kernels you need to produce 256-bit optimized kernels?
+
+DG: In a way, you might need to use a different vector length in the compiler.
+
+PP: XNNPACK is also detecting x86 vs Arm, if you want to extend revec to both platform you probably have to disable that check.
+
+IR: I feel people prefer to have predictable performance, and probably emulation needs to be detected somehow. Please join discussions in FP16 repo.
+
+AB: I am advocating for a more high level approach, and maybe we should investigate standard fallbacks that were mentioned. For example encryption/description functionality that can benefit from commonly available encryption instruction.
```
Comment (akirilov-arm) [inline simd/2024/SIMD-06-21.md:84]: `encryption/decryption functionality`

File: simd/2024/SIMD-06-21.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+DG: We are looking for better developer story, just having one API for something is very restrictive, while needing to take all possible hardware features into account is a lot of overhead. We are thinking that something
```
Comment (akirilov-arm) [inline simd/2024/SIMD-06-21.md:86]: Maybe remove the second sentence, unless you have a recollection of what else has been said?

File: simd/2024/SIMD-06-21.md (hunk 1 above)
Comment (ppenzin) [reply simd/2024/SIMD-06-21.md:84]: Good catch!

**Comment by ppenzin** [comment]
@tlively I take lack of further corrections as a sign that it is :smile:

**Pull request merged**